src/metric/SampleMetric.py

```diff
@@ -1,17 +1,21 @@
+from __future__ import annotations
 import abc
 import pandas as pd
 import numpy as np
 
-from typing import Any
+from typing import Any, TYPE_CHECKING
 
 from src.core.Setupable import Setupable
 from src.dataset.Dataset import Dataset
 from src.population.Population import Population
-import src.metric.SampleMetricManager as SampleMetricManager
+
+
```

Comment:
remove one line of spacing to match other file (one line between normal imports and the "if")